catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexer.java

```diff
@@ -0,0 +1,285 @@
+/**
```

Comment:
we should take lucene out of the path since we are not writing this to a local lucene index anymore